src/styles/blocks/header.scss

```
@@ -0,0 +1,58 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin: 0 auto;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  &__list {
+    display: flex;
+    list-style-type: none;
+    padding-inline-start: 0;
+  }
+  &__link {
```

```suggestion
  text-transform: uppercase;
  
  &__list {
    display: flex;
    list-style-type: none;
    padding-inline-start: 0;
  }
  
  &__link {
```
doesn't linter trigger on it? 😁